resources/popup.html

```diff
@@ -4,13 +4,14 @@
 
     <link rel="stylesheet" href="../libs/materialize/css/materialize.min.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" >
+    <link rel="stylesheet" href="main.css">
 
     <script src="../libs/jquery-2.2.3.js"></script>
     <script src="../libs/materialize/js/materialize.min.js"></script>
 
     <script src="popup.js"></script>
 </head>
-    <body style="min-width: 300px; min-height: 300px;">
+    <body style="min-width: 200px; min-height: 200px;">
```

Comment:
But you already linked the main.css right? And then also the styling for <div class="row center"> should be added